server/src/app/config.py

```diff
@@ -4,8 +4,23 @@
 # See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
 
 
+import json
+
 PROJECT_NAME: str = 'Pyronear local webserver'
 PROJECT_DESCRIPTION: str = 'API for pyronear systems interactions on sites'
 API_BASE: str = 'api/'
 VERSION: str = "0.1.0a0"
 LOGO_URL: str = "https://github.com/pyronear/PyroNear/raw/master/docs/source/_static/img/pyronear-logo-dark.png"
+
+
+# security
+JWT_ENCODING_ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 365 * 10
+
+# to get a string like this run:
+# openssl rand -hex 32
```

Comment:
Let's not scream out to the world how to brute force our secret key :upside_down_face: 
We'll need to change how it gets generated as well then

Reply:
Okay ! I have another idea !